app/src/main/java/com/kabouzeid/gramophone/dialogs/AddDuplicateToPlaylistDialog.java

```diff
@@ -0,0 +1,93 @@
+package com.kabouzeid.gramophone.dialogs;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+import android.text.Html;
+
+import com.afollestad.materialdialogs.DialogAction;
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.kabouzeid.gramophone.R;
+import com.kabouzeid.gramophone.model.Song;
+
+import java.util.ArrayList;
+
+/**
+ * @author Karim Abou Zeid (kabouzeid)
+ */
+public class AddDuplicateToPlaylistDialog extends DialogFragment {
+
+    private int selection = 0;
+
+    private SelectionListener selListener;
+
+    public AddDuplicateToPlaylistDialog() {
+        selListener = null;
+    }
+
+    public void setSelListener(SelectionListener SelListener) {
+        this.selListener = SelListener;
+    }
+    public interface SelectionListener {
+        public void selectedOption(int selection);
+    }
+
+    @NonNull
+    public static AddDuplicateToPlaylistDialog create(Song song) {
+        ArrayList<Song> list = new ArrayList<>();
```

Comment:
You should try to see if you can get this to just be a list of IDs since there is a restriction on the max bundle size.